terraform/terraform_test.go

```diff
@@ -285,6 +285,22 @@ module.child:
     type = aws_instance
 `
 
+const testTerraformApplyModuleBoolStr = `
+aws_instance.bar:
+  ID = foo
+  foo = 1
+  type = aws_instance
+
+  Dependencies:
+    module.child
+
+module.child:
+  <no state>
+  Outputs:
+
+  leader = 1
+`
```

Comment:
It's weird to me that this lives so far away from the test. Is the idea that another test might also reference this?